WindowsServerDocs/networking/technologies/network-subsystem/net-sub-performance-tuning-nics.md

```diff
@@ -119,14 +119,15 @@ For a TCP receive window that has a particular size, you can use the following e
 
 For example, for a connection that has a latency of 10 ms, the total achievable throughput is only 51 Mbps. This value is reasonable for a large corporate network infrastructure. However, by using autotuning to adjust the receive window, the connection can achieve the full line rate of a 1-Gbps connection.
```

Comment:
Is "51Mbps" still accurate? And is it still considered "reasonable for a large corporate network infrastructure"? That confident statement makes me cringe a little bit.

Reply:
```suggestion
```